generation/WinSDK/enums.json

```diff
@@ -31894,6 +31894,525 @@
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OLEUIOBJECTPROPSA",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OLEUIOBJECTPROPSW",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "VIEW_OBJECT_PROPERTIES_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "VPF_",
+      "header": "oledlg.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "OLEUIVIEWPROPSA",
+        "parameter": "dwFlags"
+      },
+      {
+        "method": "OLEUIVIEWPROPSW",
+        "parameter": "dwFlags"
+      }
+    ]
+  },
+  {
+    "name": "FADF",
+    "type": "ushort",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "FADF_",
+      "header": "oaidl.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "SAFEARRAY",
+        "field": "fFeatures"
+      }
+    ]
+  },
+  {
+    "name": "DISPATCH",
+    "type": "ushort",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "DISPATCH_",
+      "header": "oaidl.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "interface": "ITypeInfo",
+        "method": "Invoke",
+        "field": "wFlags"
+      }
+    ]
+  },
+  {
+    "name": "IMPLTYPEFLAGS",
+    "type": "int",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "IMPLTYPEFLAG_",
+      "header": "oaidl.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "interface": "ITypeInfo",
+        "method": "GetImplTypeFlags",
+        "field": "pImplTypeFlags"
+      },
+      {
+        "interface": "ITypeInfo",
+        "method": "SetImplTypeFlags",
+        "field": "ImplTypeFlags"
+      }
+    ]
+  },
+  {
+    "name": "PARAMFLAGS",
+    "type": "ushort",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "PARAMFLAG_",
+      "header": "oaidl.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "PARAMDESC",
+        "field": "wParamFlags"
+      }
+    ]
+  },
+  {
+    "name": "IDLFLAGS",
+    "type": "ushort",
+    "flags": true,
+    "members": [
+      {
+        "name": "IDLFLAG_NONE",
+        "value": "0"
+      },
+      {
+        "name": "IDLFLAG_FIN",
+        "value": "0x1"
+      },
+      {
+        "name": "IDLFLAG_FOUT",
+        "value": "0x2"
+      },
+      {
+        "name": "IDLFLAG_FLCID",
+        "value": "0x4"
+      },
+      {
+        "name": "IDLFLAG_FRETVAL",
+        "value": "0x8"
+      }
+    ],
+    "uses": [
+      {
+        "struct": "IDLDESC",
+        "field": "wIDLFlags"
+      }
+    ]
+  },
+  {
+    "name": "NUMPARSE_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "NUMPRS_",
+      "header": "oleauto.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "NUMPARSE",
+        "field": "dwInFlags"
+      },
+      {
+        "struct": "NUMPARSE",
+        "field": "dwOutFlags"
+      }
+    ]
+  },
+  {
+    "name": "PICTYPE",
+    "type": "int",
+    "flags": true,
```

Comment:
I think this is wrong. `PICTYPE` is _not_ a flags enum. From the headers:

```c
/////////////////////////////////////////////////////////////////////////////
// PICTDESC structure

#define PICTYPE_UNINITIALIZED (-1)
#define PICTYPE_NONE            0
#define PICTYPE_BITMAP          1
#define PICTYPE_METAFILE        2
#define PICTYPE_ICON            3
#ifdef _WIN32
#define PICTYPE_ENHMETAFILE     4
#endif
```

We can see these aren't bit-aware constants. They increment by one and they use -1 as a value.